Samples/auth/Outlook-Add-in-SSO-events/README.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Use SSO with event-based activation in an Outlook add-in"
+page_type: sample
+urlFragment: outlook-add-in-sso-event-based-activation
+products:
+- office-add-ins
+- office-outlook
+- office
+languages:
+- javascript
+extensions:
+  contentType: samples
+  technologies:
+  - Add-ins
+  createdDate: 02/21/2023 10:00:00 AM
+description: "Use SSO with event-based activation in an Outlook add-in."
+---
+
+# Use SSO with event-based activation in an Outlook add-in
+
+**Applies to**: Outlook on Windows | Outlook on the web | [new Outlook on Mac](https://support.microsoft.com/office/6283be54-e74d-434e-babb-b70cefc77439)
+
+## Summary
+
+The sample shows how to use SSO to access a user's Microsoft Graph data from an event fired in an Outlook add-in. The sample illustrates the following concepts:
+
+- [Get a user access token using SSO](https://learn.microsoft.com/outlook/add-ins/authenticate-a-user-with-an-sso-token) when the [OnNewMessageCompose event](https://learn.microsoft.com/office/dev/add-ins/outlook/autolaunch) fires.
+- Implement a server REST API that uses the [Microsoft identity platform and OAuth 2.0 On-Behalf-Of flow](https://learn.microsoft.com/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow) to exchange the user's access token for a new access token with permissions to the users profile on Microsoft Graph.
+- Use the [Microsoft Graph API](https://developer.microsoft.com/graph/docs/api-reference/v1.0/resources/onedrive) to get user profile data such as display name and job title.
+- Construct a signature in the mail item containing the user profile data.
+
+## Register the add-in with Microsoft identity platform
+
+Use the following values for the subsequent app registration steps.
+
+| Placeholder or section | Value          |
+|------------------------|----------------|
+| `<add-in-name>`          | `outlook-event-sso-sample` |
+| `<fully-qualified-domain-name>` | `localhost:3000` |
+| Microsoft Graph permissions | profile, openid, user.read |
+
+Follow the steps in [Register an Office Add-in that uses single sign-on (SSO) with the Microsoft identity platform](https://learn.microsoft.com/office/dev/add-ins/develop/register-sso-add-in-aad-v2).
+
+> Note: The instructions have you create a redirect URI for a single-page application. This step isn't necessary for this sample because it doesn't use a fallback authentication approach if SSO fails.
```

Comment:
```suggestion
> Note: The instructions tell you to create a redirect URI for a single-page application. This step isn't necessary for this sample because it doesn't use a fallback authentication approach if SSO fails.
```